layer/filestore_unix.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
@AkihiroSuda Just to clarify - are you saying -do- read/write the files on Unix for now?